sh/folder.go

```diff
@@ -6,28 +6,41 @@ import (
 	"github.com/wal-g/tracelog"
 	"golang.org/x/crypto/ssh"
 	"io"
+	"fmt"
+	"path/filepath"
 )
 
 type Folder struct {
 	client SftpClient
 	path string
 }
 
+const (
+	Port = "SSH_PORT"
+	Password = "SSH_PASSWORD"
```

Comment:
Now it's plain text password in the env variable?

Reply:
want to use ssh-key in future but now just "as is" :3